.travis.yml

```diff
@@ -19,11 +19,13 @@ env:
 cache:
   directories:
     - $HOME/.composer/cache
+    - solr_downloads
 
 before_install:
   - composer global require "hirak/prestissimo:^0.3"
-  - curl -O http://archive.apache.org/dist/lucene/solr/${SOLR_VERSION}/solr-${SOLR_VERSION}.tgz
-  - tar -xzf solr-${SOLR_VERSION}.tgz
+  - mkdir -p solr_downloads && ls solr_downloads/
```

Comment:
mkdir is not required, it's done by travis if needed.
have a look at the current master for a working implemenation.